code/stable/gamephysics/Website/GamePhysics_SRS.html

```diff
@@ -2159,7 +2156,19 @@ <h3>Instance Models</h3>
                     </tr>
                   </table>
                 </div>
-                <p class="paragraph"></p>
+                <div id="IM:transMotDeriv">
+                  <div class="subsubsubsection">
+                    <h4>
+                      Detailed derivation of force on the translational motion of a set of 2D rigid bodies:
+                    </h4>
+                    <p class="paragraph">
+                      We may calculate the total acceleration of rigid body <em>j</em> by calculating the derivative of it's velocity with respect to time.
+                    </p>
+                    \[{\mathbf{a}_{j}}=\frac{\,d{\mathbf{v}_{j}}\left(t\right)}{\,dt}\]
+                    <p class="paragraph">Performing the derivative, we obtain:</p>
+                    \[{\mathbf{a}_{j}}=\mathbf{g}+\frac{{\mathbf{F}_{j}}\left(t\right)}{{m_{j}}}\]
+                  </div>
+                </div>
```

Comment:
This will likely need to be improved. I am just trying to move the original derivation component of the equation of the transMot Instance Model to the derivation portion of the Instance Model. @smiths If it's alright, can you please give this a look?